CONTRIBUTING.md

```diff
@@ -41,16 +41,7 @@ If you have a production issue, please [contact Firebase support][support] who w
 
 Before you submit your issue, try searching [past issues][archive], [StackOverflow][stackoverflow], and the [Firebase Google Group][firebase-google-group] for issues similar to your own. You can help us to maximize the effort we spend fixing issues, and adding new features, by not reporting duplicate issues. 
 
-If your issue appears to be a bug, and hasn't been reported, open a new issue. Providing the following information will increase the chances of your issue being dealt with quickly:
-
-* **Description of the Issue** - if an error is being thrown a non-minified stack trace helps
-* **Motivation for or Use Case** - explain why this is a bug for you
-* **Related Issues** - has a similar issue been reported before?
-* **Environment Configuration** - is this a problem with Node.js, or only a specific browser? Is this only in a specific version of the SDK?
-* **Reproduce the Error** - provide a live example (like [JSBin][jsbin]), a Github repo, or an unambiguous set of steps
-* **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point to what might be causing the problem (line of code or commit)
-
-There is an issue template provided to help capture all of this information. Following the template will also help us to route your issue to the appropriate teams faster, helping us to better help you!
+If you encounter an issue that appears to be a bug that has not been reported before, please open a new issue. When filling out the new issue report form, be sure to include as much information as possible, such as reproduction steps, the error message you received, and any screenshots or other relevant data. The more context you can provide, the better we will be able to understand the issue, route it to the appropriate team, and provide you with the help you need.
```

Comment:
should we link to the template from here?

Reply:
Sounds like a good idea, +1